python/mujincontrollerclient/controllerclientbase.py

```diff
@@ -676,10 +679,23 @@ def FlushAndDownloadFile(self, filename, timeout=5):
             raise ControllerClientError(response.content.decode('utf-8'))
         return response
 
+    def FlushAndHeadFile(self, filename, timeout=5):
+        """Flush and perform a HEAD operation on given filename to retrieve metadata.
+
+        :return: a dict containing "modified (datetime.datetime)" and "size (int)"
+        """
+        response = self._webclient.Request('HEAD', '/file/download/', params={'filename': filename}, timeout=timeout)
+        if response.status_code != 200:
+            raise ControllerClientError(response.content.decode('utf-8'))
+        return {
+            'modified': datetime.datetime(*email.utils.parsedate(response.headers['Last-Modified'])[:6]),
+            'size': int(response.headers['Content-Length']),
+        }
+
```

Comment:
@ziyan, do we still need this after the new webstack changes?

Reply:
Yes.